models/pathology_nuclei_segmentation_classification/configs/inference.json

```diff
@@ -0,0 +1,150 @@
+                "a_max": 255.0,
+                "b_min": 0.0,
+                "b_max": 1.0,
+                "clip": true
+            }
+        ]
+    },
+    "data_list": "$[{'image': image} for image in glob.glob(os.path.join(@dataset_dir, '*.png'))]",
+    "dataset": {
+        "_target_": "Dataset",
+        "data": "@data_list",
+        "transform": "@preprocessing"
+    },
+    "dataloader": {
+        "_target_": "DataLoader",
+        "dataset": "@dataset",
+        "batch_size": "@batch_size",
+        "shuffle": false,
+        "num_workers": "@num_cpus",
+        "pin_memory": true
+    },
+    "inferer": {
+        "_target_": "SlidingWindowHoVerNetInferer",
+        "roi_size": "@patch_size",
+        "sw_batch_size": "@sw_batch_size",
+        "overlap": "$1.0 - float(@out_size) / float(@patch_size)",
+        "padding_mode": "constant",
+        "cval": 0,
+        "sw_device": "@device",
+        "device": "@device",
+        "progress": true,
+        "extra_input_padding": "$((@patch_size - @out_size) // 2,) * 4"
+    },
+    "postprocessing": {
+        "_target_": "Compose",
+        "transforms": [
+            {
+                "_target_": "FlattenSubKeysd",
+                "keys": "pred",
+                "sub_keys": [
+                    "horizontal_vertical",
+                    "nucleus_prediction",
+                    "type_prediction"
+                ],
+                "delete_keys": true
+            },
+            {
+                "_target_": "HoVerNetInstanceMapPostProcessingd",
+                "sobel_kernel_size": 3,
+                "marker_threshold": 0.7,
+                "marker_radius": 2
+            },
+            {
+                "_target_": "HoVerNetNuclearTypePostProcessingd"
+            },
+            {
+                "_target_": "FromMetaTensord",
+                "keys": [
+                    "image"
+                ]
+            },
+            {
+                "_target_": "SaveImaged",
+                "keys": "instance_map",
+                "meta_keys": "image_meta_dict",
+                "output_ext": "png",
+                "output_dir": "@output_dir",
+                "output_postfix": "instance_map",
+                "output_dtype": "uint8",
+                "separate_folder": false
+            },
+            {
+                "_target_": "SaveImaged",
+                "keys": "type_map",
+                "meta_keys": "image_meta_dict",
+                "output_ext": "png",
+                "output_dir": "@output_dir",
+                "output_postfix": "type_map",
+                "output_dtype": "uint8",
+                "separate_folder": false
+            }
+        ]
+    },
+    "handlers": [
+        {
+            "_target_": "CheckpointLoader",
+            "load_path": "$os.path.join(@bundle_root, 'models', 'model.pt')",
+            "map_location": "@device",
+            "load_dict": {
+                "model": "@network"
```

Comment:
the large file used doesn't have "mode" as load dict..
before raising the PR.. please make sure basic testing is done for infer/train workflows